utils/platform.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package utils
+
+import (
+	"fmt"
+	v08 "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
+	"github.com/kiegroup/kogito-serverless-operator/constants"
+	"github.com/pkg/errors"
+)
+
+func GetCustomConfig(platform v08.KogitoServerlessPlatform) (map[string]string, error) {
+	customConfig := make(map[string]string)
+	if platform.Namespace == "" {
+		return nil, errors.New(fmt.Sprintf("Unable to retrieve the namespace from platform %s", platform.Name))
+	}
+	customConfig[constants.CUSTOM_NS_KEY] = platform.Namespace
```

Comment:
I am not sure but, keeo registry secret in the operator side might not be needed, once it is created in the namespace ocp/k8s will use it automatically.

unless the reason to get it here is print some warning message or something else.